openeogeotrellis/ml_model.py

```diff
@@ -17,9 +23,88 @@ def write_assets(self, directory: str) -> Dict:
         :return: STAC assets dictionary: https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#assets
         """
         directory = pathlib.Path(directory).parent
-        filename = str(pathlib.Path(directory) / "mlmodel.model")
-        self._model.save(gps.get_spark_context() , filename)
+        filename = str(pathlib.Path(directory) / "randomforest.model")
+        self._model.save(gps.get_spark_context(), filename)
         return {filename:{"href":filename}}
 
+    def get_model(self):
+        return self._model
+
     def save_ml_model(self, directory: str) -> Dict:
-        return self.write_assets(directory)
\ No newline at end of file
+        return self.write_assets(directory)
+
+    def create_flask_response(self) -> Response:
```

Comment:
I'm not sure this method is necessary. Will it be used at the moment?

Reply:
This is only used when fit_class_random_forest is called with a synchronous request, right? Then it won't be used yet.